.gha-reversemap.yml

```diff
@@ -49,15 +49,15 @@ azure/setup-helm:
   tag: v4.3.0
   tag-url: https://github.com/azure/setup-helm/tree/v4.3.0
 docker/login-action:
-  sha: 184bdaa0721073962dff0199f1fb9940f07167d1
-  sha-url: https://github.com/docker/login-action/commit/184bdaa0721073962dff0199f1fb9940f07167d1
-  tag: v3.5.0
-  tag-url: https://github.com/docker/login-action/tree/v3.5.0
+  sha: 74a5d142397b4f367a81961eba4e8cd7edddf772
+  sha-url: https://github.com/docker/login-action/commit/74a5d142397b4f367a81961eba4e8cd7edddf772
+  tag: v3
```

Comment:
Do not make the tag generic. The intent here is to use a tag that identifies one immutable version.